storybook/src/foundation/design-tokens/aspect-ratio.docs.mdx

```diff
@@ -0,0 +1,22 @@
+{/* @license CC0-1.0 */}
```

Comment:
Can you also add (probably very similar) docs to Utilities/CSS/Aspect Ratio?

Reply:
I was considering adding either this or both. It does feel as if it is doubling. Currently, we don't do that (have token and CSS docs). Any thoughts about that @VincentSmedinga?